golangci-lint/action.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'golangci-lint'
+description: |
+  The action runs golangci-lint and reports issues from linters.
+
+inputs:
+  args:
+    description: |
+      The arguments to pass to golangci-lint
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name:  golangci-lint ${{ inputs.args }}
+      uses: golangci/golangci-lint-action@5c56cd6c9dc07901af25baab6f2b0d9f3b7c3018 #v2.5.2
+      with:
+        args: ${{ inputs.args }}
```

Comment:
I don't understand what we'd get by wrapping this existing action?

Generally I've been using composite actions to encapsulate a bunch of bash or other logic, but here I think folks should just use this action directly.

Reply:
manage every golangci-lint args or version from a common place, maybe?

Reply:
Historically, I've seen this mostly configured through `.golangci.yaml`

I'm gonna close this one for now, but definitely let us know if you have more ideas for things to encapsulate!